S/squashfs_tools/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "squashfs_tools"
+version = v"4.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/plougher/squashfs-tools.git", "c570c6188811088b12ffdd9665487a2960c997a0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd squashfs-tools/squashfs-tools/
+sed -i 's|#XZ_SUPPORT = 1|XZ_SUPPORT = 1|' Makefile
+sed -i 's|#LZO_SUPPORT = 1|LZO_SUPPORT = 1|' Makefile
+sed -i 's|#LZ4_SUPPORT = 1|LZ4_SUPPORT = 1|' Makefile
+sed -i 's|#ZSTD_SUPPORT = 1|ZSTD_SUPPORT = 1|' Makefile
+make
```

Comment:
Why not doing
```suggestion
make XZ_SUPPORT=1 LZO_SUPPORT=1 LZ4_SUPPORT=1 ZSTD_SUPPORT=1 -j${nproc}
```
?

Reply:
No reason, I just followed their building instructions.